generic/MaplGenericComponent.F90

```diff
@@ -16,11 +16,14 @@ module mapl_MaplGenericComponent
    use mapl_MaplGrid
    use mapl_RunEntryPoint
    use mapl_EntryPointVector
+   use mapl_ESMF_Interfaces
+   use mapl_CallbackMap
    implicit none
    private
 
    public :: MaplGenericComponent
    public :: get_grid
+   public :: MAPL_AddMethod
 
    procedure(), pointer :: user_method => null()
```

Comment:
```suggestion
   procedure(), pointer :: user_method => null()
```
I can't remember why this is a module variable.  Unrelated to this PR, but should be fixed I think.

Reply:
It appears not to be needed. The line can and will be removed.